subscriptions/models.py

```diff
@@ -333,3 +333,16 @@ def __str__(self):
         return '{},{}:{}/{}'.format(
             self.send_date, self.messageset.short_name,
             self.estimate_subscriptions, self.estimate_identities)
+
+
+@python_2_unicode_compatible
+class ResendRequest(models.Model):
+
+    """ Resend Request from user, used to trigger a resend.
+    """
+    received_at = models.DateTimeField(auto_now_add=True)
+    subscription = models.ForeignKey(Subscription, on_delete=models.CASCADE)
+    outbound = models.UUIDField(null=True)
```

Comment:
I'm wondering if we should also store a link to the message that we resent, in case we need to go back and see what language, messageset, sequence_number, etc we sent to them.

Reply:
It now saves the message, it would be good to have the extra info later on. Ready for another look.